src/app/app/router/effects/router.effects.ts

```diff
@@ -65,6 +66,26 @@ export class RouterEffects {
 		map(([action, router]: [SelectCaseAction | SaveCaseAsSuccessAction, IRouterState]) => new NavigateCaseTriggerAction(action.payload.id))
 	);
 
+	@Effect()
+	loadDefaultCase$: Observable<any> = this.actions$.pipe(
+		ofType(CasesActionTypes.LOAD_DEFAULT_CASE),
+		withLatestFrom(this.store$.select(routerStateSelector)),
+		mergeMap(([action, router]: [(SelectDilutedCaseAction), IRouterState]) => {
+			if (router.linkId) {
+				return this.casesService.storageService.get(this.casesService.linksConfig.schema, router.linkId).pipe(
```

Comment:
make a function in `casesService` that return link from storage instead of public the `storageService`